### ISSUE TITLE
Extra space in plural aliases

### ISSUE THREAD

**Issue by OfficialDonut** [SkriptLang/Skript#2004]
### Description

A bunch of aliases have an extra space before the 's' in the plural syntax:  
https://hastebin.com/uqaxikuyet.sql

For example:
```
command /test:
    trigger:
        set {_test} to bricks
```
This code gives the error: `Can't understand this expression: 'bricks' (bug.sk, line 3: set {_test} to bricks')`

But if the space before the 's' is added -> `set {_test} to brick s` it will not error

### Server Information
* **Server version/platform:** Paper 1.13.2      
* **Skript version:** 2.3.6

**Comment by TheBentoBox** [comment]
Good find. That's due to aliases which are written like this:
```
[({rail straight}|{rail curve})] [normal] [minecart] (rail|track) [block]¦s = minecraft:rail
```
I was told this was fine (for the plurality marker to follow an optional block), and this *is* the ideal way for us to write them aliases-side too, but clearly it isn't working properly. I could fix this in the actual aliases but realistically this should be fixed in the alias parser itself -- when it is excluding an optional block that is immediately followed by a plurality marker, it should trim all white space before the optional block for that alias result.

This is pretty important since it means a ton of normal aliases aren't working which should, i.e. any of them that have an optional block immediately preceding the plurality marker.

**Comment by FranKusmiruk** [comment]
Should be fixed for the next release.

**Comment by TheBentoBox** [comment]
Fixed in the 2.4 alphas.
